app/assets/javascripts/dashboard/views/dash/explorative_annotations_view.js

```diff
@@ -0,0 +1,354 @@
+    if (!this.state.showArchivedTracings) {
+      return (
+        <div>
+          <a href={controller.trace(typ, id).url}>
+            <i className="fa fa-random" />
+            <strong>trace</strong>
+          </a>
+          <br />
+          <a href={jsRoutes.controllers.AnnotationIOController.download(typ, id).url}>
+            <i className="fa fa-download" />
+            download
+          </a>
+          <br />
+          <a href="#" onClick={() => this.finishOrReopenTracing("finish", tracing)}>
+            <i className="fa fa-archive" />
+            archive
+          </a>
+          <br />
+        </div>
+      );
+    } else {
+      return (
+        <div>
+          <a href="#" onClick={() => this.finishOrReopenTracing("reopen", tracing)}>
+            <i className="fa fa-folder-open" />
+            reopen
+          </a>
+          <br />
+        </div>
+      );
+    }
+  };
+
+  getCurrentTracings(): Array<APIAnnotationType> {
+    return this.state.showArchivedTracings
+      ? this.state.archivedTracings
+      : this.state.unarchivedTracings;
+  }
+
+  handleSearch = (event: SyntheticInputEvent): void => {
+    this.setState({ searchQuery: event.target.value });
+  };
+
+  renameTracing(tracing: APIAnnotationType, name: string) {
+    const tracings = this.getCurrentTracings();
+
+    const newTracings = tracings.map(currentTracing => {
+      if (currentTracing.id !== tracing.id) {
+        return currentTracing;
+      } else {
+        return update(currentTracing, { name: { $set: name } });
+      }
+    });
+
+    this.setState({
+      [this.state.showArchivedTracings ? "archivedTracings" : "unarchivedTracings"]: newTracings,
+    });
+
+    const url = `/annotations/${tracing.typ}/${tracing.id}/name`;
+    const payload = { data: { name } };
+
+    Request.sendJSONReceiveJSON(url, payload).then(response => {
+      Toast.message(response.messages);
+    });
+  }
+
+  archiveAll = () => {
+    if (!confirm("Are you sure you want to archive all explorative annotations?")) {
+      return;
+    }
+
+    const unarchivedAnnoationIds = this.state.unarchivedTracings.map(t => t.id);
+    Request.sendJSONReceiveJSON(
+      jsRoutes.controllers.AnnotationController.finishAll("Explorational").url,
+      {
+        method: "POST",
+        data: {
+          annotations: unarchivedAnnoationIds,
+        },
+      },
+    ).then(data => {
+      Toast.message(data.messages);
+
+      this.setState({
+        archivedTracings: this.state.unarchivedTracings.concat(this.state.archivedTracings),
+        unarchivedTracings: [],
+      });
+    });
+  };
+
+  renderTable() {
+    return (
+      <Table
+        dataSource={Utils.filterWithSearchQuery(
+          this.getCurrentTracings(),
+          ["id", "name", "dataSetName", "contentType"],
+          this.state.searchQuery,
+        )}
+        rowKey="id"
+        pagination={{
+          defaultPageSize: 50,
+        }}
+        className="clearfix"
```

Comment:
The search bar above the table floats right which is why the float needed to be cleared here. Otherwise, the search bar collided with the table.